test/langtools/tools/javac/patterns/Unnamed.java

```diff
@@ -0,0 +1,238 @@
+import java.util.Objects;
+
+/**
+ * @test /nodynamiccopyright/
```

Comment:
there is no golden file for this test so full copyright will be needed

Reply:
Also, it is a custom that the comment with `@test` is before any imports, AFAIK.